spiketoolkit/comparison/groundtruthcomparison.py

```diff
@@ -222,22 +223,31 @@ def count_well_detected_units(self, **kargs):
         """
         return len(self.get_well_detected_units(**kargs))
 
-    def get_false_positive_units(self):
+    def get_false_positive_units(self, min_redundant_agreement=0.3):
         """
         Return units list of "false positive units" from tested_sorting.
         
         "false positive units" ara defined as units in tested that
         are not matched at all in GT units.
         
         Need exhaustive_gt=True
+
+        Parameters
+        ----------
+        min_redundant_agreement: float (default 0.3)
+            The minimum agreement between gt and tested units
+            that are best match to be counted as "redundant" units and not "bad".
+
         """
         assert self.exhaustive_gt, 'false_positive_units list is valid only if exhaustive_gt=True'
-        fake_ids = []
+        best_match = list(self._unit_map12.values())
+        false_positive_ids = []
         unit2_ids = self.sorting2.get_unit_ids()
         for u2 in unit2_ids:
-            if self._best_match_units_21[u2] == -1:
-                fake_ids.append(u2)
-        return fake_ids
+            if u2 not in best_match:
+                if u2 not in self.get_redundant_units(min_redundant_agreement):
```

Comment:
This can be slow in a loop.

Reply:
I'll use what you suggested (checking best match 21 and 12)

Reply:
I am not sure if we must split in 3.
I propose make a quite and dirty fix for the paper now and open a PR for more discussion about: FPunit/bad/redundant/intermediateFP...